### PR TITLE
Milestone: Implemented sending and receiving data from/to the Leaderboard API

### PR DESCRIPTION
**Pull request by Roseokpe**
I implemented the sending and receiving function.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,23 +13,18 @@ <h1>Leaderboard</h1>
...
+       </ul>
       </div>
     </section>
     <section>
       <h4>Add Your Score</h4>
       <form action="">
-        <input type="text" placeholder="Your name" name="name">
-        <input type="text" placeholder="Your score" name="name">
+        <input type="text" placeholder="Your name" name="name" class="uName">
+        <input type="text" placeholder="Your score" name="name" class="uScore">
```
Comment (ahmetbozaci) [inline src/index.html:27]: ### OPTIONAL
- [x] It will be good if you change the score type to the number. Right now ı can add a text as a score.

**Review by ahmetbozaci — APPROVED** [review]
# 🥳 🎉🥇 💯  💥 🎆 🎈 STATUS: APPROVED 🥳 🎉  🥇  💯  💥   🎆 🎈 🎉 



@Roseokpe  ✋🏼 
#### Your project is excellent. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.
___ 

### 💯 Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- App is working well ı can add scores and by clicking the refresh button ı can see the scores.✔️
- All functions are arrow functions.✔️

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
aproved


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: src/index.html (hunk 1 above)
Comment (Roseokpe) [reply src/index.html:27]: Thank you

**Review by BertrandConxy — APPROVED** [review]
Hi @Roseokpe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- clean UI
- project works fine
- no linter errors
- correct git-flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**